.changeset/migrate-1713465834005.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-adr': patch
+'@backstage/plugin-adr-backend': patch
+'@backstage/plugin-adr-common': patch
```

Comment:
These three are repeated from the changeset above. I'm on mobile now but maybe worth double checking through these